adoc/architecture-description.adoc

```diff
@@ -0,0 +1,316 @@
+
+In addition, {productname} simplifies the platform operator’s experience, with everything you need to get up and running quickly, and to manage the environment effectively in production. It provides:
+
+* A complete container execution environment, container run time (CRI-O), and container image registries (registry.suse.com).
+* Application ecosystem support with {sle} container base images, and access to tools and services offered by SUSE Ready for CaaS Platform partners and the Kubernetes community.
+* Enhanced datacenter integration features that enable you to plug Kubernetes into new or existing infrastructure, systems, and processes.
+* End-to-End security, implemented holistically across the full stack including networking policies via Cilium, `PodSecurityPolicies` and {kube} RBAC.
+* Advanced platform management that simplifies platform installation, configuration, re-configuration, monitoring, maintenance, updates, and recovery.
+* Enterprise hardening including comprehensive interoperability testing, support for thousands of platforms, and world-class platform maintenance and technical support.
+
+You can deploy {productname} onto physical servers or use it on virtual machines. After deployment, it is immediately ready to run and provides a highly-scalable cluster.
+
+{productname} inherits benefits of SUSE Linux Enterprise and uses tools and technologies well-known to system administrators such as cloud-init, AutoYaST and Terraform.
+
+{productname} uses the Btrfs file system with the following characteristics:
+
+* The root file system and its snapshots are read-only.
+* Sub-volumes for data sharing are read-write.
+* {productname} introduces overlays of the /etc directories used by cloud-init and Salt.
+
+For more information, including a list of the various components which make up {productname} please refer to the Release Notes on https://www.suse.com/releasenotes/.
+
+[WARNING]
+====
+This diagram needs to be reviewed and recreated with more clean looking graphics
+====
+
+image::architecture-caasp-components.jpg[SUSE CaaSP Components, width=100%,pdfwidth=80%]
+
+=== Product Release Cycle
+
+Product releases are aligned with {sle} releases and {kube}. Major version jumps
+are determined by the {sle} release cycle. Minor version jumps by the quarterly {kube} releases.
+Individual maintenance updates with receive patchlevel numbering.
+
+Versioning scheme: `x.y.z`
+
+* x - Base OS version
+* y - Kubernetes version
+* z - Patchlevel (incl. new features !)
+
+[%autowidth.stretch]
+|===
+|Version |Base OS |{kube} version |Patchlevel
+
+|4.0.0
+|{sle} 15 SP1
+^|1.15
+^|0
+
+|4.0.1
+|{sle} 15 SP1
+^|1.15
+^|1
+
+|...
+|
+|
+|
+
+|4.1.0
+|{sle} 15 SP1
+^|1.16
+^|0
+
+|...
+|
+|
+|
+
+|5.0.0
+|{sle} 15 SP2
+^|tbd
+^|0
+|===
+
+
+== Supported Platforms
+
+* SUSE OpenStack Cloud 8
+* VMWare ESXi 6.0.0 Update 3
+* Bare Metal
+** KVM
+** x86_64
```

Comment:
I think it's a little weird to mix architectures with platforms here. It would be good to have a clear statement on what architectures are supported on every platform. If only `x86_64` is supported right now I think something along the lines of:

```
== Supported platforms

* SUSE OpenStack Cloud 8
* VMWare ESXi 6.0.0 Update 3
* Bare Metal
```

```
== Supported architectures

* x86_64
```

would be better. Also, KVM is something different than bare metal. Bare metal would imply no hypervisor at all, whereas KVM is a virtualization platform. I think for the time being KVM can be out of the list.

Reply:
>  It would be good to have a clear statement on what architectures are supported on every platform.

Fully agree!